pages/release-process.md

```diff
@@ -144,6 +144,72 @@ When the vote passes on a release candidate, follow the steps below to complete
 
  8.  Send an email to `dev@fluo.apache.org` announcing new release.
 
+### Test a Fluo release
+
+Below are two common testing strategies to verify a Fluo release candidate:
+
+1. Run the integration tests of projects that depend on Fluo (using the release candidate jars)
+
+1. Run example Fluo applications (using the release candidate tarball)
+
+These strategies are described below with step-by-step instructions.
+
+#### Run integration tests
+
+A Fluo release can be tested by running the integration tests of projects that use Fluo.
+
+1. Before running integration tests, you'll need the following information:
+    * Release version (example: `1.2.0`)
+    * Staging repo (example: `https://repository.apache.org/content/repositories/orgapachefluo-1023/`)
+        * Note: The staging repo changes with each release candidate.
```

Comment:
Too wordy. Better to use a system property in the below profile, and have the user set it on the command line.
This can become:
```
1. Set the version number: $ export RCVERSION=1.2.0
2. Set the staging repository URL: $ export STAGINGREPO=https://...orgapachefluo-1023/
```